src/routes/yourplace/index.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'preact';
+import Heading from '../../components/Heading';
+import Button from '../../components/Button/index';
+import BlueHint from '../../components/BlueHint';
+import ToggleBox from '../../components/ToggleBox';
+import PlaceNumber from '../../components/PlaceNumber';
+
+import style from './style.scss';
+
+export default class YourPlace extends Component {
+  state = {
+    placeNumber: 1.337,
+    placeIsAvailable: false,
+  };
+
+  togglePlaceAvailability = () => {
+    this.setState(prevState => ({ placeIsAvailable: !prevState.placeIsAvailable }));
+  };
+
+  renderPlaceAvailable = placeNumber => (
+    <div>
+      <Heading> Twoje miejsce: </Heading>
```

Comment:
This leading and trailing whitespace seems weird. I think you should handle it with styling